src/EnergyPlus/SingleDuct.cc

```diff
@@ -1791,8 +1791,11 @@ namespace SingleDuct {
 		//Do a check and make sure that the max and min available(control) flow is
 		//  between the physical max and min while operating.
 		SysInlet( SysNum ).AirMassFlowRateMaxAvail = min( Sys( SysNum ).AirMassFlowRateMax, Node( InletNode ).MassFlowRateMaxAvail );
-		SysInlet( SysNum ).AirMassFlowRateMinAvail = min( max( Node( OutletNode ).MassFlowRateMin, Node( InletNode ).MassFlowRateMinAvail ), SysInlet( SysNum ).AirMassFlowRateMaxAvail );
-
+		if ( Sys( SysNum ).SysType_Num == SingleDuctConstVolReheat ) { // AirTerminal:SingleDuct:ConstantVolume:Reheat
```

Comment:
@mjwitte I would like to make a new issue to deal with ideal load plus VAV case.